common/thorhelper/thorxmlwrite.cpp

```diff
@@ -246,7 +246,10 @@ void CommonXmlWriter::outputXmlns(const char *name, const char *uri)
 {
     StringBuffer fieldname;
     if (!streq(name, "xmlns"))
-        fieldname.append("xmlns:");
```

Comment:
This change isn't necessary, or actually correct.  Should probably be an exception if there is a colon in the xmlns.. see comment below.